### DISCUSSION TITLE
Chiar does not start

### DISCUSSION THREAD

**Discussion by callingcard1973** [Chia-Network/chia-blockchain#8232] [Support]
Hello all, 
It seems Windows does not start. 

Windows 7, 64 bit. It installed correctly but does not work. 

What do you think I should check ? 

Thank you all.

**Comment by loppefaaret** [comment]
Windows 7 is not supported any longer - you need a python version that is not directly compatible with windows 7

**Comment by BrandtH22** [comment]
@callingcard1973 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
